test/tests.js

```diff
@@ -0,0 +1,210 @@
+/* eslint-env node, mocha */
+const expressFactory = require("express");
+const sri4node = require("sri4node");
+const sleep = require('await-sleep');
+
+const sri4nodeConfigFactory = require("./context/config");
+
+const testPartyAttachmentsAfterUpdateMod =  require("./testPartyAttachmentsAfterUpdate");
+
+const port = 5000;
+const base = `http://localhost:${port}`;
+
+const httpClientMod = require("./httpClient.js");
+const httpClient = httpClientMod.httpClientFactory(base);
+
+const { info, error } = require("../js/common");
+
+describe("sri4node-attachments : ", () => {
+  /** @type {import("sri4node").TSriServerInstance} */
+  let sri4nodeServerInstance;
+  let server;
+
+  before(async () => {
+    try {
+      const app = expressFactory();
+      const sriConfig = await sri4nodeConfigFactory();
+      sriConfig.description = 'config of sri4node-attachments';
+      sri4nodeServerInstance = await sri4node.configure(app, sriConfig);
+      app.set("port", port);
+      server = app.listen(port, () => {
+        info(`Node app is running at localhost:${port}`);
+      });
+    } catch (err) {
+      error("Unable to start server.");
+      error(err);
+      error(err.stack);
+    }
+  });
+
+  after(async () => {
+    // enable this to keep the server running for inspection
+    // await new Promise(() => {});
```

Comment:
It would also be handy to be able to keep them running in between tests (with an extra config param?) in case you're working on single test and need to re-run it repeatedly. In that case we would also need to clean up the data in an after hook.

Reply:
might be future improvement